tests/__init__.py

```diff
@@ -1,15 +1,14 @@
 import glob
-import os.path
+import pathlib
 import subprocess
 import sys
 
 import pytest
 
 from vulture import core
 
-DIR = os.path.dirname(os.path.abspath(__file__))
-REPO = os.path.dirname(DIR)
-WHITELISTS = glob.glob(os.path.join(REPO, "vulture", "whitelists", "*.py"))
+REPO = pathlib.Path(__file__).resolve().parents[1]
+WHITELISTS = glob.glob(str(REPO / "vulture" / "whitelists" / "*.py"))
```

Comment:
I think it would be nice to use `Path.glob` here.

Reply:
Had missed that. Will change.